analysis/paper/paper.Rmd

```diff
@@ -316,7 +317,7 @@ The first detail we see here is that overall, the proportions of articles citing
 
 The second detail is that we do not see a significant increase over time, unlike the ecology journals that steeply increase over time. Both archaeology and the other sciences we saw on the previous slide show an increase in citing R at around 2013-2014, we do not yet see much of an upward trend in the archaeology journals. Indeed, the first citations of R in several archaeology journals only occured in the last two years. The plot on the upper right of this slide shows a statistical test of change over time for articles in JAS. With a low r-squared value and high p-value for the linear model, we conclude that there is no evidence for an increase in citations over time in this journal.  
```

Comment:
Yes, so we don't see a huge growth in this in archaeology as we see in ecology. That is a fair comment. I would argue that in archaeology we're seeing the beginning, or first signs, of the huge growth that we've seen in archaeology. That is my hypothesis. We can test it in 10 years from now. 

The data that we show in this presentation shows that there has been a major change in some sciences, towards to use of R. We speculate that we may be seeing the beginnings of these changes in archaeology. And that this is a good change that we should try to accelerate. 

Does that make sense?

Reply:
Yes, I see your point and it'd be great if things developed this way! Let's phrase it a bit more explicit, like you've explained here, if that's ok with you?